apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -218,20 +218,23 @@ type EBPFFlowFilter struct {
 	TCPFlags string `json:"tcpFlags,omitempty"`
 
 	// SourcePorts defines the source ports to filter flows by.
-	// To filter a single port, set a single port as an integer value. For example sourcePorts: 80.
-	// To filter a range of ports, use a "start-end" range, string format. For example sourcePorts: "80-100".
+	// To filter a single port, set a single port as an integer value. For example, sourcePorts: 80.
+	// To filter a range of ports, use a "start-end" range, string format. For example, sourcePorts: "80-100".
```

Comment:
nit but sometimes it's mentionned `range, string format.` and sometimes `range in string format.`

It feels more fluent with the 2nd option to me.